controllers/volumereplicationgroup_controller.go

```diff
@@ -1146,6 +1151,62 @@ func (v *VRGInstance) updateVRGLastGroupSyncTime() {
 	v.instance.Status.LastGroupSyncTime = leastLastSyncTime
 }
 
+func (v *VRGInstance) updateVRGLastGroupSyncDuration() {
+	var maxLastSyncDuration *metav1.Time
+
+	for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
+
+		// If any protected PVC reports nil, report that back (no sync duration available)
+		if protectedPVC.LastSyncDuration == nil {
+			maxLastSyncDuration = nil
+
+			break
+		}
+
+		if maxLastSyncDuration == nil {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+
+			continue
+		}
+
+		if protectedPVC.LastSyncDuration != nil && protectedPVC.LastSyncDuration.After(maxLastSyncDuration.Time) {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+		}
+	}
+
+	v.instance.Status.LastGroupSyncDuration = maxLastSyncDuration
+}
+
+func (v *VRGInstance) updateLastGroupSyncBytes() {
```

Comment:
I don't understand this last group sync bytes (or I don't understand the way it is implemented).  I thought of it as cumulative sync bytes of all PVCs. If that assumption is correct, then why line 1186 turns it off if one of the PVCs didn't sync?

Reply:
I have updated he code